pootle/apps/pootle_language/models.py

```diff
@@ -33,7 +33,9 @@ class LiveLanguageManager(models.Manager):
     def get_queryset(self):
         return super(LiveLanguageManager, self).get_queryset().filter(
             translationproject__isnull=False,
-            project__isnull=True,
```

Comment:
No, we do no need that anymore. That bit is to exclude languages that are used as `source_language` in any project. With this PR we are trying to expose those.